packages/commands/generate/src/index.ts

```diff
@@ -29,14 +25,8 @@ export default defineCommand<{}, CliFlags>(() => {
         },
       });
     },
-    async handler(args) {
-      if (args.backend) {
-        await generateUsingPlugins(args);
-      } else if (args.db) {
-        await createDBResources({});
```

Comment:
We have moved DB to the templates itself as most of the users want to control when database migration happens and CLI was to restrictive - could not handle ability to connect to remote databases very well. 

New change gives developers more flexibility to specify custom scalars, filter operations etc. that will be hard to represent for CLI